src/dataprotection/azext_dataprotection/manual/Manifests/AzureDatabaseForPostgreSQL.py

```diff
@@ -13,6 +13,30 @@
   "allowedRestoreTargetTypes": [ "AlternateLocation", "RestoreAsFiles" ],
   "itemLevelRecoveyEnabled": false,
   "supportSecretStoreAuthentication": true,
+  "backupVaultPermissions": [
+    {
+      "roleDefinitionName": "Reader",
+      "assignedOn": "AzureDatabaseForPostgreSQL",
+      "accessMethod": [
+        "properties",
+        "data_source_info",
+        "resource_id"
+      ],
+      "truncate": 9

```

Comment:
truncate is very specific for the postgresql workload. Does not make much sense in the manifest. We can instead tackle it in the code using the datasourcetype.